README.md

```diff
@@ -264,6 +264,7 @@ Currently, the following customizations are possible:
 |--------------------|-----------------------------------------------------------------------------------------|
 | otp\_path          | Path to the OTP installation                                                            |
 | deps\_dirs         | List of directories containing dependencies. It supports wildcards.                     |
+| apps\_dirs         | List of directories containing project applications. It supports wildcards.             |
```

Comment:
Should we add what the default value is (i.e. `["."]`)?

Reply:
I thought about it, but we don't do it for any other case. Let's improve the config docs separately from this PR.